test/extension-developer-e2e/setup.sh

```
@@ -102,6 +94,7 @@ reg_bundle_push_tag="${local_registry_host}/${reg_bundle_path}"
     --resource --controller && \
   export OPERATOR_SDK="${operator_sdk}" && \
   make generate manifests && \
+  sed -i -e 's/$(CONTAINER_TOOL) build/$(CONTAINER_TOOL) build --provenance=false/' Makefile && \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)